src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Cms.Module/.template.config/template.json

```diff
@@ -23,8 +23,8 @@
           "description": "Target net5.0"
         }
       ],
-      "replaces": "netcoreapp3.1",
-      "defaultValue": "netcoreapp3.1"
+      "replaces": "$(TemplateTargetPackageFramework)",
```

Comment:
@agriffard something has gone slightly of with the merge here. Can you check that what you expect to be in these template modules is still there ok?